src/main/java/uk/gov/ons/ctp/response/action/endpoint/ActionEndpoint.java

```diff
@@ -243,6 +245,22 @@ public ActionDTO updateAction(
     }
   }
 
+  /**
+   * PUT to re-run the aborted Actions for a specified actionId and to reschedule them .
+   *
+   * @param actionIds Action Id of the Action/s to re run
+   * @return Success for the re ran actions
+   * @throws CTPException if update operation fails
+   */
+  @RequestMapping(value = "/rerun", method = RequestMethod.PUT)
+  public ResponseEntity<List<UUID>> rerunAction(
+      @RequestParam(value = "actionId") @Valid List<UUID> actionIds) throws CTPException {
```

Comment:
What is the `@Valid` doing here?

Reply:
Its there to verify that the the list passed in is not null etc